### ISSUE TITLE
Build targeting PaleMoon browser?

### ISSUE THREAD

**Issue by frmariam** [lazarus-recovery/lazarus_addon#25]
Hi there.
Lazarus is till one of my must have extensions, and one of the reasons I still prefer FF forked projects that continue to support XUL extensions.

Palemoon  has very recently cut all official support for any extension that does not directly target it.
I believe the reasons are to try to motivate the community devs to maintain the old extensions and also to make a clean break from the FF brand.

If you are inclined to post an updated version of Lazarus 2.3.1 I (an many PM users) would be very grateful:
https://forum.palemoon.org/viewtopic.php?f=19&t=16785

Best regards

**Comment by XJDHDR** [comment]
It is highly unlikely that any of the devs are going to do this. The bulk of the work done on this repo was to port Lazarus to the WebExtensions API, not to continue maintaining the XUL version. And even that work hasn't really resulted in anything. [The consensus](https://github.com/lazarus-recovery/lazarus_addon/issues/21#issuecomment-614338365) seems to be that there is no point in doing any further work on this addon and that everyone might as well migrate to Form History Control II instead. Not just because the codebase is old but because Lazarus was designed for websites of it's era.

So in short, if you want Lazarus to continue working on Palemoon, you'll probably have to do it yourself because it's unlikely anyone here will do so. TBH, it's not that hard to do this; Moonchild's forum post tells you how to do it. It's just a matter of unpacking the extension's XPI file (which is just a renamed ZIP file), follow the instructions and then re-pack the modified files into a new XPI.
